graphrag/query/cli.py

```diff
@@ -179,3 +175,184 @@ async def run_streaming_search():
     # NOTE: we return the response and context data here purely as a complete demonstration of the API.
     # External users should use the API directly to get the response and context data.
     return response, context_data
+
+
+def _resolve_parquet_files(
```

Comment:
Can we extract this function to an util? Can we also add an optional parameter to pass a list of the files one is interested on? 
To provide an example, I'm definitely needing this for incremental indexing, so I can load a few cherry picked outputs :)

Reply:
Another suggestion is, the function seems a bit long, can we refactor into multiple functions?